database/nosql/cassandra/README.md

```diff
@@ -0,0 +1,56 @@
+# Apache Cassandra
```

Comment:
Please consider noting that allowing a general user to use this is potentially dangerous as the start script can be edited.

Reply:
That's a good point. I will add a note as soon as I have time to do so.

Reply:
Security notice has been added in the security section of the README